[2025-06-06T18:54:52.876+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=etl_music_pipeline/run_id=manual__2025-06-06T18:49:44.518448+00:00/task_id=transform_spotify permission to 509
[2025-06-06T18:54:52.876+0000] {logging_mixin.py:188} INFO - Failed to change /opt/***/logs/dag_id=etl_music_pipeline/run_id=manual__2025-06-06T18:49:44.518448+00:00/task_id=transform_spotify permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=etl_music_pipeline/run_id=manual__2025-06-06T18:49:44.518448+00:00/task_id=transform_spotify'
[2025-06-06T18:54:52.894+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=etl_music_pipeline/run_id=manual__2025-06-06T18:49:44.518448+00:00/task_id=transform_spotify permission to 509
[2025-06-06T18:54:52.894+0000] {logging_mixin.py:188} INFO - Failed to change /opt/***/logs/dag_id=etl_music_pipeline/run_id=manual__2025-06-06T18:49:44.518448+00:00/task_id=transform_spotify permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=etl_music_pipeline/run_id=manual__2025-06-06T18:49:44.518448+00:00/task_id=transform_spotify'
[2025-06-06T18:54:52.918+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_music_pipeline.transform_spotify manual__2025-06-06T18:49:44.518448+00:00 [queued]>
[2025-06-06T18:54:52.923+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_music_pipeline.transform_spotify manual__2025-06-06T18:49:44.518448+00:00 [queued]>
[2025-06-06T18:54:52.924+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2025-06-06T18:54:52.932+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): transform_spotify> on 2025-06-06 18:49:44.518448+00:00
[2025-06-06T18:54:52.936+0000] {standard_task_runner.py:60} INFO - Started process 1663 to run task
[2025-06-06T18:54:52.938+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_music_pipeline', 'transform_spotify', 'manual__2025-06-06T18:49:44.518448+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpk8xie1av']
[2025-06-06T18:54:52.940+0000] {standard_task_runner.py:88} INFO - Job 70: Subtask transform_spotify
[2025-06-06T18:54:52.969+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=etl_music_pipeline/run_id=manual__2025-06-06T18:49:44.518448+00:00/task_id=transform_spotify permission to 509
[2025-06-06T18:54:52.969+0000] {logging_mixin.py:188} INFO - Failed to change /opt/***/logs/dag_id=etl_music_pipeline/run_id=manual__2025-06-06T18:49:44.518448+00:00/task_id=transform_spotify permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=etl_music_pipeline/run_id=manual__2025-06-06T18:49:44.518448+00:00/task_id=transform_spotify'
[2025-06-06T18:54:52.970+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_music_pipeline.transform_spotify manual__2025-06-06T18:49:44.518448+00:00 [running]> on host 3dc3cd419f05
[2025-06-06T18:54:53.019+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_music_pipeline' AIRFLOW_CTX_TASK_ID='transform_spotify' AIRFLOW_CTX_EXECUTION_DATE='2025-06-06T18:49:44.518448+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-06T18:49:44.518448+00:00'
[2025-06-06T18:54:53.020+0000] {etl_pipeline.py:25} INFO - [START] Task: transform_spotify
[2025-06-06T18:54:53.020+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 199, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 216, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/etl_pipeline.py", line 26, in wrapper
    result = func(*args, **kwargs)
  File "/opt/airflow/dags/etl_pipeline.py", line 33, in wrapper
    func(*args, **kwargs)
TypeError: transform_spotify() got an unexpected keyword argument 'conf'
[2025-06-06T18:54:53.028+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=etl_music_pipeline, task_id=transform_spotify, execution_date=20250606T184944, start_date=20250606T185452, end_date=20250606T185453
[2025-06-06T18:54:53.035+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 70 for task transform_spotify (transform_spotify() got an unexpected keyword argument 'conf'; 1663)
[2025-06-06T18:54:53.070+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-06-06T18:54:53.084+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
